java/device/doc/how_to_build_a_java_app_from_scratch.md

```diff
@@ -20,12 +20,15 @@ See [Prepare your development environment][devbox-setup] for information about p
 <a name="createproject"/>
 ## Create the project
 
-On your command line, execute the following command to create a new, empty maven project:
+On your command line, execute the following command to create a new, empty maven project in a suitable location on your development machine:
 
 ```
-mvn archetype:generate -DgroupId=com.mycompany.app -DartifactId=iot-device -DarchetypeArtifactId=maven-archetype-quickstart -DinteractiveMode=false
+mvn archetype:generate -DgroupId=com.mycompany.app -DartifactId=iot-device 
+-DarchetypeArtifactId=maven-archetype-quickstart -DinteractiveMode=false
```

Comment:
It seems that this was broken down into 2 lines, I believe copy pasting 2 lines in a shell window will actually not work. Could you fix this to be on one line?